.gitlab/repository.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "readme",
+  "url": "https://gitlab.com/thefriending/readme",
```

Fix namespace of repository

```suggestion
  "url": "https://gitlab.com/friending/readme",
```